core/src/test/kotlin/org/evomaster/core/database/insertion/postgres/PostgresInsertValueTest.kt

```diff
@@ -0,0 +1,118 @@
+package org.evomaster.core.database.insertion.postgres
+
+import org.evomaster.client.java.controller.db.SqlScriptRunner
+import org.evomaster.core.KGenericContainer
+import org.junit.jupiter.api.*
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy
+import java.sql.Connection
+import java.sql.DriverManager
+import java.sql.SQLException
+
+class PostgresInsertValueTest {
+
+    companion object {
+
+        @JvmStatic
+        protected lateinit var connection: Connection
+
+        private val POSTGRES_VERSION: String = "14"
+
+        private val postgres = KGenericContainer("postgres:$POSTGRES_VERSION")
+                .withEnv("POSTGRES_HOST_AUTH_METHOD", "trust")
+                .withExposedPorts(5432)
+                //https://www.postgresql.org/docs/current/auth-trust.html
+                .withEnv("POSTGRES_HOST_AUTH_METHOD", "trust")
+
+        @BeforeAll
+        @JvmStatic
+        fun initClass() {
+            postgres.start()
+            val host = postgres.containerIpAddress
+            val port = postgres.getMappedPort(5432)!!
+
+            val url = "jdbc:postgresql://$host:$port/postgres"
+
+            /*
+             * A call to getConnection()  when the postgres container is still not ready,
+             * signals a PSQLException with message "FATAL: the database system is starting up".
+             * The following issue describes how to avoid this by using a LogMessageWaitStrategy
+             * https://github.com/testcontainers/testcontainers-java/issues/317
+             */
+            postgres.waitingFor(LogMessageWaitStrategy().withRegEx(".*database system is ready to accept connections.*\\s").withTimes(2))
```

Comment:
possibly this timeout needs to be increased? and, is it `s` or `ms`?

Reply:
It seems to be passing now in the CI and in my system.